scripts/installation/relock_dependencies.sh

```diff
@@ -1,48 +1,25 @@
 #!/usr/bin/env bash
 
-set -e
-
 # can change output file names with rebuild_pipenv.sh <prefix>
 PREFIX=${1:-requirements}
 
-read -p "Update your pip and setuptools? (type y or Y) " -n 1 -r
-echo
-if [[ $REPLY =~ ^[Yy]$ ]]
-then
-    pip install --upgrade pip
-    pip install --upgrade setuptools
-fi
-
+# these steps might fail, but that's okay.
 echo "Removing existing lock files..."
 pipenv --rm
 rm -f Pipfile.lock
 rm -f $PREFIX.txt
 rm -f dev-$PREFIX.txt
 
-read -p "Remove system pip and pipenv cache? (type y or Y) " -n 1 -r
-echo
-if [[ $REPLY =~ ^[Yy]$ ]]
-then
-    echo "Removing pip and pipenv system cache..."
-    rm -r ~/.cache/pip ~/.cache/pipenv
-fi
+echo "Removing pip and pipenv system cache..."
+rm -r ~/.cache/pip ~/.cache/pipenv
 
-echo "Rebuilding dependency locks..."
+# start enforcing failures
+set -e
 
-echo "Bulding dev-$PREFIX.txt..."
+echo "Building Development Requirements"
 pipenv lock --clear --pre --requirements --dev > dev-$PREFIX.txt
 
-echo "Building $PREFIX.txt..."
+echo "Building Standard Requirements"
 pipenv lock --clear --pre --requirements > $PREFIX.txt
 
-echo "Installing nucypher into new virtual enviorment..."
-pipenv run pip install -e . -r dev-$PREFIX.txt
-
-echo "Checking nucypher CLI entry point..."
```

Comment:
You don't want to still check that the `nucypher` CLI can be run? Is that because it will be checked as part of the makefile instead? (See my comment in `Makefile`)

Reply:
I think the name of this script was a bit of a misnomer since it contains installation.